packages/mattermost-redux/src/actions/threads.ts

```diff
@@ -94,21 +75,21 @@ export function handleThreadArrived(dispatch: DispatchFunc, getState: GetStateFu
         type: ThreadTypes.RECEIVED_THREAD,
         data: {
             thread,
-            team_id: teamId,
+            team_id: teamId || currentTeamId,
```

Comment:
When would we ever receive a websocket event for a thread where we didn't also receive a team ID?

Reply:
DM/GM `thread_updated` websocket events don't have a teamId since DM/GM channels travel with the current-team.